Source/WebCore/html/parser/HTMLElementStack.h

```diff
@@ -92,6 +92,8 @@ class HTMLElementStack {
     ElementRecord* furthestBlockForFormattingElement(Element&) const;
     ElementRecord* topmost(ElementName) const;
 
+    bool containsTemplateElement() const { return m_templateElementCount; }
```

Comment:
This struck me as odd initially, but I guess in general we're comfortable casting from integers to booleans.

Reply:
yeah, we generally don't explicitly cast in those cases. We could use `!!` too but here the function declaration makes it pretty obvious so I think it's good as is.